src/main/kotlin/org/rust/ide/inspections/import/AutoImportFix.kt

```diff
@@ -138,6 +138,12 @@ class AutoImportFix(element: RsElement, private val context: Context) :
             return Context(GENERAL_PATH, candidates)
         }
 
+        fun findApplicableContext(pat: RsPatBinding): Context? {
+            val importContext = ImportContext2.from(pat, ImportContext2.Type.AUTO_IMPORT) ?: return null
+            val candidates = ImportCandidatesCollector2.getImportCandidates(importContext, pat.referenceName)
```

Comment:
Since we dropped old auto import code, can we remove `2` suffix now? Of course, not in this PR